packages/bun-types/globals.d.ts

```diff
@@ -14,6 +14,12 @@ declare module "bun" {
       ? {}
       : import("node:stream/web").DecompressionStream;
 
+    /**
+     * Compression formats supported by Bun's CompressionStream and DecompressionStream.
+     * Bun extends the standard web formats with brotli and zstd support.
+     */
+    type CompressionFormat = "gzip" | "deflate" | "deflate-raw" | "brotli" | "zstd";
```

Comment:
I wonder if we can extend the global type instead of needing to redeclare it all